scss/mixins/_mixins.scss

```diff
@@ -225,28 +225,44 @@
 
 @mixin fd-link($color: $fd-color--action) {
   @include fd-var-color("color", $fd-color--action, --fd-color-action-1);
-  text-decoration: none;
   display: inline-block;
   transition: all $fd-animation--speed ease-in;
-  border-bottom-style: solid;
-  border-bottom-width: 1px;
-  @include fd-var-color("border-bottom-color", $fd-color--action, --fd-color-action-1);
+  text-decoration: none;
+  position: relative;
+  &::after {
+    content: '';
+    width: 100%;
+    position: absolute;
+    left: 0;
+    bottom: 1px;
```

Comment:
Looking at the `fd-link` instances from `master`, I think this value should be `bottom: 0;`.